### PR TITLE
Karthiknagaraj okta 227648

### PR DESCRIPTION
**Pull request by karthikmallekavnagaraj-okta**
Resolves OKTA-227648 : Minor fixes to Group rules API doc.

Description:
1. Removing rule description which says still in beta.
2. Create and update operations on the group rules which supports multiple target groups. as of now.

* [OKTA-227648](https://oktainc.atlassian.net/browse/OKTA-227648)

### CODE REVIEW

**Comment by karthikmallekavnagaraj-okta** [comment]
Content Changes can be previewed at https://github.com/okta/okta-developer-docs/blob/00933730fc6e32036ad9b8e5910d6511f252339d/packages/%40okta/vuepress-site/docs/api/resources/groups/index.md.

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "chalk": "^2.4.2",
     "cross-env": "^5.2.0",
     "if-env": "^1.0.4",
-    "mocha": "^5.2.0",
```
Comment (amoghvasekar-okta) [inline package.json]: I dont think we should be updating this file

File: yarn.lock [hunk 2]
```diff
@@ -4,23 +4,23 @@
 
 "@babel/code-frame@^7.0.0":
   version "7.0.0"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
```
Comment (amoghvasekar-okta) [inline yarn.lock]: Same here. I dont think we should update this file.
You can revert to the original file using git reset

**Comment by motikagan-okta** [comment]
@karthikmallekavnagaraj-okta, the docs repo went through a big directory structure migration after you opened your pull request, so there’s now a merge conflict. Would you like me to go in and resolve the conflict, and push up the changes to the branch?

File: package.json (hunk 1 above)
Comment (karthikmallekavnagaraj-okta) [reply package.json]: Reverted the file back to master version.

**Pull request merged**